src/components/textfieldControl/css/TextfieldSlider.css

```diff
@@ -23,78 +23,83 @@ http://brennaobrien.com/blog/2014/05/style-input-type-range-in-every-browser.htm
 */
 
 .fl-textfieldSlider input[type=range].fl-slider {
-  -webkit-appearance: none;
-  flex-grow: 1;
-  margin: 0 1rem;
+    -webkit-appearance: none;
+    flex-grow: 1;
+    margin: 0 1rem;
 }
 
 /* webkit browsers */
 .fl-textfieldSlider input[type=range].fl-slider::-webkit-slider-runnable-track {
-  height: 1rem;
-  cursor: pointer;
-  animate: 0.2s;
-  background: #dad6d3;
-  border-radius: 0.3rem;
-  border: 1px solid #b3b3b3;
+    animation-duration: 0.2s;
+    background: #dad6d3;
+    border: 1px solid #b3b3b3;
+    border-radius: 0.3rem;
+    cursor: pointer;
+    height: 1rem;
 }
+
 .fl-textfieldSlider input[type=range].fl-slider::-webkit-slider-thumb {
-    height: 1.5rem;
-    width: 1.5rem;
+    -webkit-appearance: none;
     background-color: #fff;
+    background-image: linear-gradient(to bottom left, #fff 46%, #e9eaea 73%);
     border: 1px solid #b3b3b3;
     border-radius: 2rem;
-    box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
-    background-image: linear-gradient(right top, #fff 46%, #e9eaea 73%);
-    -webkit-appearance: none;
+    box-shadow: 4px 2px 3px rgba(0, 0, 0, 0.3);
+    height: 1.5rem;
     margin-top: -0.3rem;
+    width: 1.5rem;
 }
 
 /* Firefox */
 .fl-textfieldSlider input[type=range].fl-slider::-moz-range-track {
-  height: 1rem;
-  cursor: pointer;
-  animate: 0.2s;
-  background: #dad6d3;
-  border-radius: 0.3rem;
-  border: 1px solid #b3b3b3;
+    animation-duration: 0.2s;
+    background: #dad6d3;
+    border: 1px solid #b3b3b3;
+    border-radius: 0.3rem;
+    cursor: pointer;
+    height: 1rem;
 }
+
 .fl-textfieldSlider input[type=range].fl-slider::-moz-range-thumb {
-    height: 1.5rem;
-    width: 1.5rem;
     background-color: #fff;
+    background-image: linear-gradient(to bottom left, #fff 46%, #e9eaea 73%);
     border: 1px solid #b3b3b3;
     border-radius: 2rem;
-    box-shadow: 4px 2px 3px rgba(0,0,0,0.3);
-    background-image: linear-gradient(right top, #fff 46%, #e9eaea 73%);
+    box-shadow: 4px 2px 3px rgba(0, 0, 0, 0.3);
+    height: 1.5rem;
+    width: 1.5rem;
 }
 
 /* ms browsers */
 .fl-textfieldSlider input[type=range].fl-slider::-ms-track {
-  height: 1rem;
-  animate: 0.2s;
-  background: transparent;
-  border-color: transparent;
-  border-width: 0.5rem 0;
-  color: transparent;
+    animation-duration: 0.2s;
+    background: transparent;
+    border-color: transparent;
+    border-width: 0.5rem 0;
+    color: transparent;
+    height: 1rem;
 }
+
 .fl-textfieldSlider input[type=range].fl-slider::-ms-fill-lower {
```

Comment:
someday we should review all these vendor prefixes.

Reply:
Ideally we could write unprefixed CSS and use Autoprefixer (see: [FLUID-6566](https://issues.fluidproject.org/browse/FLUID-6566)).